pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -279,7 +288,13 @@ func validateVCenterVersion(validationCtx *validationContext, fldPath *field.Pat
 		validationCtx.Client.ServiceContent.About.Version, validationCtx.Client.ServiceContent.About.Build)
 
 	switch {
+	// case: vCenter < 7.0.2
+	case csiConstraint.Check(vCenterVersion):
+		// case: < 7.0.2
+		logrus.Warnf("VMware vSphere 7 is end of service as of 10/2/2025. Current vCenter version: %s, build: %d", vCenterVersion.String(), build)
+		allErrs = append(allErrs, field.Required(fldPath, detail))
 	case eolConstraints.Check(vCenterVersion):
+		// case: >= 7.0.0 < 8.0.0
```

Comment:
we may need to either update the comment or update the case. It appears that the first case and this one overlap.